DevAcademy/Scenes/Places/PlacesRow.swift

```diff
@@ -7,33 +7,70 @@
 
 import SwiftUI
 
-struct PlacesRow: View {
+struct PlaceDetail: View {
     let feature: Feature
     
     var body: some View {
-        HStack {
-            
-            AsyncImage(url: feature.properties.obrId1) {
-                image in
+        VStack(alignment: .leading, spacing: 16) {
+            AsyncImage(url: feature.properties.obrId1) { image in
                 image
                     .resizable()
-                    .frame(width: 60, height: 60)
-                    .aspectRatio(contentMode: .fill)
+                    .aspectRatio(contentMode: .fit)
                     .clipShape(RoundedRectangle(cornerRadius: 8))
                     .shadow(radius: 4)
-            } placeholder : {
+            } placeholder: {
                 ProgressView()
             }
-            VStack(alignment : .leading) {
-                Text(feature.properties.nazev)
-                    .font(.title)
-                    .fontWeight(.semibold)
-                    .lineLimit(1)
-                Text(feature.properties.druh.rawValue)
-                    .foregroundColor(.secondary)
-                    .font(.subheadline)
+            .frame(maxHeight: 200)
+            
+            Text(feature.properties.nazev)
+                .font(.title)
+                .fontWeight(.semibold)
+            
+            Text(feature.properties.druh.rawValue)
+                .foregroundColor(.secondary)
+                .font(.subheadline)
+            
+            Spacer()
+        }
+        .padding()
```

Comment:
Tady by neměl být potřeba Spacer. Ten se hodí na odtlačení dvou prvků od sebe, ale pokud potřebuješ napozicovat view, tak bych zkusil pomocí frame.
```suggestion
        }
        .frame(maxHeight: .infinity, alignment: .top)
        .padding()
```